src/main/java/com/example/gsheets/client/GoogleCredentialAPI.java

```diff
@@ -0,0 +1,261 @@
+        final NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+        return new Sheets.Builder(HTTP_TRANSPORT, JSON_FACTORY, getCredentials())
+                .setApplicationName(APPLICATION_NAME)
+                .build();
+    }
+
+//    private static List<List<Object>> getIdList() throws IOException, GeneralSecurityException {
+//        System.out.println("getIdList");
+//        final String spreadRange = "A2:A";
+//
+//        List<List<Object>> ids = initializeSheet()
+//                .spreadsheets()
+//                .values()
+//                .get(spreadId, spreadRange)
+//                .execute()
+//                .getValues();
+//        return ids;
+//    }
+
+    //    private static BatchGetValuesResponse getIdListBatch() throws IOException, GeneralSecurityException {//getbyId
+//        List<String> ranges = Arrays.asList("A:D");
+//        BatchGetValuesResponse readResult = initializeSheet()
+//                .spreadsheets()
+//                .values()
+//                .batchGet(spreadId)
+//                .setRanges(ranges)
+//                .execute();
+//        /*   //getValues.get(0);
+//         * [
+//         *  ["id","first","second","last"],
+//         *  ["8f335c2e-9a9d-43a5-be10-4bc9a703CRIS","Cris","Bord","Gerson"],
+//         *  ["8f335c2e-9a9d-43a5-be10-4bc9a70Carlos","Carlos","Robinson","Gers"],
+//         *  ["8f335c2e-9a9d-43a5-be33-4bc9a70Hover","Hover","Rob","Jeack"],
+//         *  ["8f335c2e-9a2d-43a5-be10-4bc9a7034Berns","Berns","Garrison","Johes"],
+//         *  ["8f445c2e-9a9d-43a5-be30-4bc9a703Homer","Homer","Herb","Hibert"],
+//         *  [],
+//         *  ["8f333x2e-1a2d-43a5-be99-4bc9a703Devi","Devi","Gar","John"]
+//         * ]
+//         *
+//         *  */
+//        return readResult;
+//    }
+    private static List<List<Object>> getDataSheet() throws IOException, GeneralSecurityException {
+        List<String> ranges = Arrays.asList("A:D");
+        return initializeSheet()
+                .spreadsheets()
+                .values()
+                .batchGet(spreadId)
+                .setRanges(ranges)
+                .execute()
+                .getValueRanges()
+                .get(0)
+                .getValues();
+    }
+
+    public static void main(String[] args) throws GeneralSecurityException, IOException {
+//        var ar = Arrays.asList(
+//                Arrays.asList(1, 2, 3, 4),
+//                Arrays.asList(5, 6, 7, 8),
+//                Arrays.asList(9, 10)
+//        );
+//        Integer target = 9;
+//
+//        OptionalInt index = IntStream.range(0, ar.size())
+//                .filter(i -> ar.get(i).contains(target))
+//                .findFirst();
+//        System.out.println(index.getAsInt());
+
+        delete("8f335c2e-9a9d-43a5-be10-Robert");
+    }
+
+    public static Student update(String id, Student student) throws IOException, GeneralSecurityException {
```

Comment:
why is it static?

Reply:
Already fix this and rewrite without static